mods/d2k/maps/ordos-01a/map.yaml

```diff
@@ -0,0 +1,104 @@
+MapFormat: 11
+
+RequiresMod: d2k
+
+Title: Ordos 01a
+
+Author: Westwood Studios
+
+Tileset: ARRAKIS
+
+MapSize: 32,28
+
+Bounds: 2,2,28,24
+
+Visibility: MissionSelector
+
+Categories: Campaign
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: Random
+	PlayerReference@Creeps:
+		Name: Creeps
+		NonCombatant: True
+		Faction: Random
+		Enemies: Ordos, Harkonnen
+	PlayerReference@Ordos:
+		Name: Ordos
+		Playable: True
+		LockFaction: True
+		Faction: ordos
+		LockColor: True
+		Color: B3EAA5
+		Enemies: Harkonnen, Creeps
+	PlayerReference@Harkonnen:
+		Name: Harkonnen
+		LockFaction: True
+		Faction: harkonnen
+		LockColor: True
+		Color: FE0000
+		Enemies: Ordos
```

Comment:
There is no `Creeps` in the Harkonnen Enemies line, but there is one in the Atreides 01a Mission ([LinkToLine](https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/maps/atreides-01a/map.yaml#L46)). 
I think without it the Harkonnen are more "focused" to fight their enemies, so I prefer it. Still I would like to hear your opinion about it.